airflow/providers/dbt/cloud/provider.yaml

```diff
@@ -57,6 +57,13 @@ dependencies:
   - asgiref
   - aiohttp>=3.9.2
 
+additional-extras:
+  # pip install apache-airflow-providers-dbt-cloud[openlineage]
+  - name: openlineage
+    description: Install compatible OpenLineage dependencies
+    dependencies:
+      - apache-airflow-providers-openlineage>=1.7.0
```

Comment:
I don't think this is right approach(?)
We can wrap the needed import with try/catch and raise informative message that explain to the user that openlineage provider is not installed.

Reply:
We have dedicated exception for such cases: `AirflowOptionalProviderFeatureException`

See example
https://github.com/apache/airflow/blob/1c9a6609f36a6fabddfd6d3858cca049d4088668/airflow/providers/google/cloud/transfers/adls_to_gcs.py#L28-L32

Reply:
@eladkal just added 464a5a02184861a0b3de788b2cbdc31b7d34ec5b , please, let me know if it looks better and if you'd like any further changes!

Reply:
It's still good to have it as additional dependency here, but @eladkal was right we should make the runtime check as well as this "additional dependency" is more of a hint than anything else (it's not enforced)